exporters/otlp/otlpmetric/otlpmetricgrpc/example_test.go

```diff
@@ -68,14 +68,16 @@ func Example_insecure() {
 		}
 	}()
 
-	meter := global.Meter("test-meter")
+	// TODO Bring Back Global package
+	// meter := global.Meter("test-meter")
+	meter := pusher.Meter("test-meter")
```

Comment:
```suggestion
	meter := pusher.Meter("go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc_test")
```

We should be modeling good practices here, and everywhere we obtain a meter/tracer.